### PR TITLE
Prepare for PHPUnit 6

### PR DESCRIPTION
**Pull request by hboomsma**
After these changes the project is compatible with PHPUnit 6.0.
The symfony/phpunit-bridge however is still not compatible, so
the version requirement in the composer.json is not elevated to
^6.0 yet. As soon as symfony fixes their bridge this can be done.

### CODE REVIEW

File: phpunit.xml.dist [hunk 1]
```diff
@@ -10,11 +10,12 @@
   -->
 
 <phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:noNamespaceSchemaLocation="http://schema.phpunit.de/5.2/phpunit.xsd"
+         xsi:noNamespaceSchemaLocation="http://schema.phpunit.de/6.0/phpunit.xsd"
          backupGlobals="false"
          bootstrap="vendor/autoload.php"
          colors="true"
          enforceTimeLimit="true"
+         beStrictAboutTestsThatDoNotTestAnything="false"
```
Comment (theofidry) [inline phpunit.xml.dist]: is that change necessary?

Comment (hboomsma) [inline phpunit.xml.dist]: Not yet, but in PHPUnit 6.0 you will get a lot of risky tests and exit code 1 when you have no explicit asserts in them. This maintains 5.x behaviour for now.

But since I did not update the constraint yet, it is not strictly necessary.

Comment (hboomsma) [inline phpunit.xml.dist]: I can remove it, with no problem, just let me know what is preferable.

Comment (theofidry) [inline phpunit.xml.dist]: Ah I see it becomes strict by default in 6.0. Fine then :)

**Comment by theofidry** [comment]
Thanks!
